saleor/checkout/steps.py

```diff
@@ -136,6 +136,7 @@ def __init__(self, request, storage, cart,
                  default_address=None):
         self.cart = cart
         address_data = storage.get('address', {})
+        self.billing_address = default_address
```

Comment:
Rename `default_address` to `shipping_address` as it's no longer a mere default.